interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,8 +3,19 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
Did you accidentally convert from this approach into the other where elements are included directly in the page? This one feels more reusable.